setup.py

```diff
@@ -414,6 +415,37 @@ def has_html(self):
     sub_commands.append(('install_html', has_html))
 
 
+class CheckTestsErrors(Command):
```

Comment:
Nice idea to do this check via Python code.  A lot easier than from the Windows command line!